### ISSUE TITLE
VoiceOver not reading custom emoji in timelines

### ISSUE THREAD

**Issue by pitermach** [metabolist/metatext#36]
**Describe the bug**

While browsing through any timeline, if either the display name or post include custom server emoji, other clients speak their shortcode, which is usually descriptive enough to figure out what it is. currently, Metatext strips them out completely, to the point that if you use the "characters" option on the VoiceOver rotor, in the space where the emoji should be, you just hear silence. Interestingly enough, on the details screen for a toot, VoiceOver announces the emoji correctly.
**To Reproduce**

With VoiceOver on, open any timeline and find a toot which either contains custom server emoji, or whose author's display name contains them.

**Expected behavior**

In the place where the emoji appears, VoiceOver should read its shortcode, IE :blobcat_smile:

**(please complete the following information):**
 - Device: iPhone SE
 - OS: iOS 14.5.1
 - App Version 1.5.0

**Additional context**

Funnily enough, this bug could  also be adapted into a  pretty useful feature. Twitterrific has a setting to filter out emoji from display names for VoiceOver, and a lot of people really like to have it turned on, so I can see a similar toggle being popular in Metatext as well, though then it would have to filter out both regular unicode Emoji as well as the server ones. Having said that the default behavior should definitely be that custom emoji get read out.
